src/js/components/Books/Book.scss

```diff
@@ -0,0 +1,16 @@
+.Books{
+    background-color: #555;
+}
+
+.bookCardFrame{
+    min-height: 150px;
+}
+
+.minHeight150{
+    min-height: 150px;
+}
+    
+.noPaddMar{
+    padding: 3px !important;
```

Comment:
Avoid using important.
Seek to understand the layer of elements that hinder your padding and inspect those classes or parent elements to make changes without using important

Reply:
Thank you, I looked into it and removed it, I better controlled the flow of the algorithms.

Reply:
Thank you, I looked into it and removed it, I better controlled the structures.